dwave_networkx/generators/chimera.py

```diff
@@ -65,6 +73,18 @@ def chimera_graph(m, n=None, t=None, create_using=None, node_list=None, edge_lis
         If True, node labels are 4-tuples, equivalent to the chimera_index
         attribute as below.  In this case, the `data` parameter controls the
         existence of a `linear_index attribute`, which is an int.
+    check_node_list : bool (optional, default False)
+        If True, the node_list elements are checked for compatibility with
```

Comment:
I think it's either ``True`` (Python literal so formatted as code) or "true" (English adjective). Also below.

the ``node_list`` elements

Reply:
Changing True and False to code everywhere leads to a lot of red in the marked up text, not sure if it is better to prefer 'is true' and 'is false' where possible?